sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/implementation/CertificatePropertiesPage.java

```diff
@@ -4,16 +4,16 @@
 package com.azure.security.keyvault.certificates.implementation;
 
 import com.azure.core.http.rest.Page;
-import com.azure.security.keyvault.certificates.models.CertificateBase;
+import com.azure.security.keyvault.certificates.models.CertificateProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.List;
 
 /**
- * A page of Azure App Configuration {@link CertificateBase} resources and a link to get the next page of
+ * A page of Azure App Configuration {@link CertificateProperties} resources and a link to get the next page of
```

Comment:
A page of Azure App Configuration  ?

Reply:
Not part of public API, doesn't matter.
But Copy pasta, fixed.